Sources_v3/StreamChatUI/ChatChannel/MessageComposerSendButton.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class MessageComposerSendButton: ChatSquareButton {
+    // MARK: Underlying types
+    
+    public enum Mode {
+        case new, edit
+    }
+    
+    // MARK: - Properties
+    
+    public var mode: Mode = .new {
+        didSet {
+            updateContent()
```

Comment:
Please always use `updateContentIfNeeded()`. Never call it directly -> https://www.notion.so/UIConfig-and-View-Structure-bdbd703e4ba44e1c8039b43ccf17f56f

Reply:
You put a 👍 here but didn't actually fix it.